nautobot/virtualization/forms.py

```diff
@@ -649,6 +703,11 @@ def __init__(self, *args, **kwargs):
                 self.fields["untagged_vlan"].widget.add_query_param("site_id", site.pk)
                 self.fields["tagged_vlans"].widget.add_query_param("site_id", site.pk)
 
+        self.fields["parent_interface"].choices = ()
+        self.fields["parent_interface"].widget.attrs["disabled"] = True
+        self.fields["bridge"].choices = ()
+        self.fields["bridge"].widget.attrs["disabled"] = True
```

Comment:
For my education, why are these lines even needed? It seems odd that we're disabling these widgets unconditionally, and since they're DynamicModelChoiceFields, why do we need to set their `choices`?

Reply:
IMO, these fields all depend on either a device or VC master, which cannot be provided because we basically bulk editing interfaces from different device/VC masters,
The only option available is to set these fields to null.